apps/web/src/App.tsx

```diff
@@ -36,6 +36,13 @@ import { LinkVercelProjectPage } from './pages/partner-integrations/LinkVercelPr
 import { useBlueprint } from './hooks/useBlueprint';
 import { BrandPage } from './pages/brand/BrandPage';
 import { SegmentProvider } from './store/segment.context';
+import LogRocket from 'logrocket';
+import setupLogRocketReact from 'logrocket-react';
+
+if (LOGROCKET_ID) {
```

Comment:
do we want to have it enabled during development?

Reply:
I don't think so, @LetItRock do you see any benefits of it?

Reply:
I don't think it has any benefit in dev, the analytics we need from this are only in prod.